To view and process dataset

DaliaAymanAmeen opened the pull request:
Using Dask and Napari to view and process 100µm resolution human brain dataset.

DaliaAymanAmeen committed:
Using Dask and Napari to view and process 100µm resolution human brain dataset.

sofroniewn reviewed (changes requested):
Looking good, I've requested some changes - the largest one being to move the section on getting the data higher, above the lazy loading stuff

DaliaAymanAmeen committed:
Modifications to applications/brain_dataset_tutorial.md

Co-Authored-By: Nicholas Sofroniew <sofroniewn@gmail.com>

DaliaAymanAmeen committed:
Modifications to applications/brain_dataset_tutorial.md

We changed "will lazy load the image when called given, the image name as a parameter" to "will lazy load the image data when called given a path to the data".

Co-Authored-By: Nicholas Sofroniew <sofroniewn@gmail.com>

DaliaAymanAmeen committed:
Update applications/brain_dataset_tutorial.md

We changed "view and process 100µm resolution human brain large dataset" to "view and process a high-resolution human brain imaging dataset".

Co-Authored-By: Nicholas Sofroniew <sofroniewn@gmail.com>

DaliaAymanAmeen committed:
Adding python and syntax highlighting (1).

Co-Authored-By: Genevieve Buckley <30920819+GenevieveBuckley@users.noreply.github.com>

DaliaAymanAmeen committed:
Adding python and syntax highlighting (2).

Co-Authored-By: Genevieve Buckley <30920819+GenevieveBuckley@users.noreply.github.com>

DaliaAymanAmeen committed:
Adding python and syntax highlighting (3).

Co-Authored-By: Genevieve Buckley <30920819+GenevieveBuckley@users.noreply.github.com>

DaliaAymanAmeen committed:
Adding python and syntax highlighting (4).

Co-Authored-By: Genevieve Buckley <30920819+GenevieveBuckley@users.noreply.github.com>

DaliaAymanAmeen committed:
Adding python and syntax highlighting (5).

Co-Authored-By: Genevieve Buckley <30920819+GenevieveBuckley@users.noreply.github.com>

DaliaAymanAmeen committed:
Modifications applications/brain_dataset_tutorial.md (1).

Co-Authored-By: Talley Lambert <talley.lambert@gmail.com>

DaliaAymanAmeen committed:
Adding hyperlink to io.imread.

Co-Authored-By: Talley Lambert <talley.lambert@gmail.com>

DaliaAymanAmeen committed:
Modifications to brain_dataset_tutorial.md (2).

Co-Authored-By: Talley Lambert <talley.lambert@gmail.com>

DaliaAymanAmeen committed:
Modifications to applications/brain_dataset_tutorial.md (3).

Co-Authored-By: Talley Lambert <talley.lambert@gmail.com>

DaliaAymanAmeen committed:
Modifications to applications/brain_dataset_tutorial.md (4).

Co-Authored-By: Talley Lambert <talley.lambert@gmail.com>

DaliaAymanAmeen committed:
Modifications to applications/brain_dataset_tutorial.md (6).

din1881 committed:
Modified some comments suggested by the contributors.

melissawm commented:
Hi folks, can we do another pass here? If the content is ready, I can move this to napari/napari (unless @DaliaAymanAmeen wants to do it 😄 ) Thanks!

sofroniewn commented:
If you move over to the main repo @melissawm if we don't hear back from the OP then we can do another review there - it does look like a lot of the feedback got addressed. thanks!

melissawm commented:
I completely forgot to follow up on this - will do it now.

psobolewskiPhD commented:
Closing given https://github.com/napari/docs/pull/132
